CHANGELOG.md

```diff
@@ -6,7 +6,12 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic
 Versioning](https://semver.org/spec/v2.0.0.html).
 
-<!-- ## [Unreleased] -->
+## [Unreleased]
+
+### Changed
+
+- The internal `.wireit/*/state` file was renamed to `.wireit/*/fingerprint`.
```

Comment:
There will be some leftover `state` files. They are small and shouldn't do any harm.

It will cause re-executions -- but that would happen anyway even if we didn't change the filename, because when you upgrade Wireit your package lock changes, which invalidates all fingerprints anyway.

Reply:
As long as it triggers a re-execution and won't cause wireit to get into an unexpected state that's fine by me

Reply:
Yep, it will just be a useless inert file that will eventually disappear as people `git clean` etc.